libs/generator/CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+add_library(block_generator STATIC generator.cpp)
+
+if(NOT Boost_FOUND)
```

Comment:
What if Boost was found previously but with different components set?

Reply:
It is a bug, condition removed